catalog/solr/catalog-solr-external-provider/src/main/java/ddf/catalog/solr/external/SolrHttpCatalogProvider.java

```diff
@@ -263,21 +267,22 @@ private void updateServer() {
     }
 
     private CatalogProvider getProvider() {
-        if (isServerUp(getServer())) {
-            if (resolver == null) {
-                provider = new SolrCatalogProvider(getServer(),
-                        filterAdapter,
-                        solrFilterDelegateFactory);
-            } else {
-                provider = new SolrCatalogProvider(getServer(),
-                        filterAdapter,
-                        solrFilterDelegateFactory,
-                        resolver);
+        if (firstUse) {
```

Comment:
@brendan-hofmann without the `firstUse` the CatalogProvider was getting reconfigured each time it was being called and the test testCswQueryWithValidationCheckerPlugin() was failing. It looked like it was because the catalog provider didn't know about the attributes `validation-warnings` and `validation-errors` for whatever reason - maybe this logic doesn't properly reconfigure the catalog provider?